Code-4-Community/speak-for-the-trees-frontend
# Emailer: Load HTML templates from S3

## Checklist


## Why

Resolves #259 

This allows the user to choose the email template and apply it to the email they're sending. The user can make edits to the template. 

## This PR

We passed in a form instance to sendEmailForm/index.tsx from email/index.tsx. 

## Screenshots
![Screenshot 2024-03-17 at 1 02 15 PM (1)](https://github.com/Code-4-Community/speak-for-the-trees-frontend/assets/102868495/eb87d904-0d29-4c8f-ba50-cbfe6e0e46fa)


## Verification Steps

Choose the template from the select template dropdown and a template in html should appear in the email body.
